group_vars/rhcs.yml.sample

```diff
@@ -296,6 +296,9 @@ ceph_repository: rhcs
 
 #ceph_conf_key_directory: /etc/ceph
 
+# Permissions for keyring files in /etc/ceph
+#keyring_permissions: '0640'
```

Comment:
Default should be 0600 to ensure no change from current setup.

Reply:
Oh! Sorry, slipped from my tests. I'll set it to `0600`.